go.mod

```diff
@@ -0,0 +1,14 @@
+module github.com/googleapis/gnostic-grpc
+
+go 1.12
```

Comment:
If possible, can we bump this to go 1.13?

Reply:
Works for me.

Reply:
I have a PR coming up so I am gonna change it to go 1.13 in the upcoming one and merge this one.

Reply:
SGTM